astropy/stats/funcs.py

```diff
@@ -550,7 +550,7 @@ def poisson_conf_interval(
 
     **4. 'sherpagehrels'** This rule is used by default in the fitting
     package 'sherpa'. The `documentation
-    <https://cxc.harvard.edu/sherpa4.4/statistics/#chigehrels>`_ claims
+    <https://web.archive.org/web/20220126234440/https://cxc.harvard.edu/sherpa4.4/statistics/#chigehrels>`_ claims
```

Comment:
```suggestion
    <https://cxc.cfa.harvard.edu/sherpa/statistics/#chigehrels>`_ claims
```
I don't know why the link was made to a specific version, but this works if we don't want to internet archive. I'm on the fence: In principle, the internet archive has the version that the text refers to ("URL claims that...") and pointing to a generic, non-versioned link might change that. However, it is almost impossible for sherpa to change that statistic because it's been in use for so long, that would be a massive API break. It's more likely that Sherpa adds additional text to the page that elaborates further, and that would be useful for users; so I'd favor a link to the Sherpa docs.